kit/ChildSession.cpp

```diff
@@ -635,6 +636,9 @@ bool ChildSession::_handleInput(const char *buffer, int length)
         else if (tokens.equals(0, "geta11ycaretposition"))
         {
             return getA11yCaretPosition();
+        else if (tokens.equals(0, "toggletiledumping"))
+        {
+            setDumpTiles(tokens[1] == "true");
```

Comment:
Again - it cannot be that a client can swamp a server with this request in a deployed system. We need to guard this with #if ENABLE_DEBUG  =) thanks !

Reply:
Now setting dumpTiles to false during the tile dumping function